workshops/20_python_beginners_workshop/tasks/short_tasks/practice_strings&variables.md

```diff
@@ -0,0 +1,15 @@
+# Exercise 1: Strings & Variables
+
+1. Create a variable called `name` and assign your first name to that variable.
+2. Print `"Welcome to the Dungeon, xxx!"`. Use **%s** inside of your printing and put the variable `name` behind.
```

Comment:
Maybe it is a good moment to mention double quotes and single quotes.

Reply:
Done!

Reply:
Done :)